environments/development/invoke.yml

```diff
@@ -10,6 +10,9 @@ django:
   backend_domain: "surfpol-loadbalancer-dev-1467666946.eu-central-1.elb.amazonaws.com"
   frontend_domain: "surfpol-loadbalancer-dev-1467666946.eu-central-1.elb.amazonaws.com"
 
+admin:
+  allowed_ip_ranges: ["194.171.28.0/27", "194.171.16.0/27", "195.169.127.0/24", "195.169.126.0/23", "145.90.230.0/23", "145.101.60.0/23"]
```

Comment:
I would like to limit the ip ranges to eduVPN:
145.90.230.0/23
145.101.60.0/23
2001:0610:0450:50::/60
2001:610:3:2151::/60